OrbitQt/orbitsamplingreport.cpp

```diff
@@ -86,6 +89,21 @@ void OrbitSamplingReport::on_PreviousCallstackButton_clicked() {
   RefreshCallstackView();
 }
 
+void OrbitSamplingReport::OnCurrentThreadTabChanged(int current_tab_index) {
+  if (current_tab_index == -1) {
+    return;
+  }
+
+  auto treeView = m_OrbitDataViews.at(current_tab_index);
```

Comment:
Nit: Is there a particular reason for using `.at(current_tab_index)` instead of `[current_tab_index]`?

Reply:
Yes, I agree with you. It is impossible to be out of range here, so `[current_tab_index]` would be more efficient. Will change it accordingly.